### ISSUE TITLE
When using local mode, any multi-line prints to the console will not be sent back using either balena push or balena logs.

### ISSUE THREAD

**Issue by sradevski** [balena-io/balena-cli#2185]

**Comment by edorgeville** [comment]
Original forum thread: https://forums.balena.io/t/balena-push-logs-local-mode-multi-line-prints-are-not-shown/250083

**Comment by edorgeville** [comment]
Update: using the `--debug` flag gives this error for every new log line:
```
[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 138[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 140[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 157[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 172[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 111[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 187[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 231[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 138[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 151[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 155[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token â in JSON at position 9  
[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 181[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 146[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 177[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 177[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 177[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 176[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 116[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token m in JSON at position 2  
[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 188[Debug]   Dropping device log due to failed parsing: SyntaxError: Unexpected token { in JSON at position 177[Warn]

**Comment by pdcastro** [comment]
Fixed in CLI version 12.44.8 or later.
